### PR TITLE
Update pre-commit modules

### PR DESCRIPTION
**Pull request by eqrx**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -22,11 +22,6 @@ repos:
     - --autofix
   - id: trailing-whitespace
 
-- repo: https://github.com/dnephin/pre-commit-golang
-  rev: v0.3.5
-  hooks:
-  - id: go-fmt
```
Comment (thetechnick) [inline .pre-commit-config.yaml:28]: Why is this removed?
Is it now included with the version upgrade?

Comment (eqrx) [reply .pre-commit-config.yaml:28]: Hey @thetechnick, gofumpt is already a part of golangci-lint, it uses a superset of what gofmt enforces.

**Review by thetechnick — APPROVED** [review]
